app/helpers/form_helper.rb

```diff
@@ -51,7 +53,8 @@ def us_states_territories_select_tag(options = {})
       form_input_error_messages('state', options)
   end
 
-  # rubocop:disable MethodLength, Style/WordArray
+  # rubocop:disable MethodLength,WordArray
```

Comment:
I did it for consistency since we have both in the codebase.  I will switch them all over..